pkg/cli/root.go

```diff
@@ -408,31 +412,37 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 	return nil
 }
 
-func isDefaultPolicyConfig(cfg *config.Config) bool {
+func doesAuthzContainOnlyAnonymousPolicies(cfg *config.Config) bool {
+	anonymousPolicyPresent := false
 	adminPolicy := cfg.AccessControl.AdminPolicy
 
-	log.Info().Msg("checking if default authorization is possible")
+	log.Info().Msg("checking if anonymous authorization is the only type of authorization policy configured")
 
 	if len(adminPolicy.Actions)+len(adminPolicy.Users) > 0 {
-		log.Info().Msg("admin policy detected, default authorization disabled")
-
-		return false
+		log.Info().Msg("admin policy detected, anonymous authorization is not the only authorization policy configured")
```

Comment:
You still need to return false in case other policies besides anonymous policies are found.